src/integrationtest/docker/tomcat/conf.yml

```diff
@@ -0,0 +1,20 @@
+packaging:
+  add:
+    - target/addon-self-administration-${project.version}
```

Comment:
You should use the `.war` file, because the directory does not contain the customizations from the Spring Boot plug-in.